tests/.pest/snapshots/Countries/MalawiTest/it_can_calculate_ugandan_holidays.snap

```diff
@@ -1,46 +0,0 @@
-[
```

Comment:
Why is this removed?

Reply:
That looks like it has been in the wrong directory (Ugandan holidays in Malawi) for some reason. In the snapshots dir Malawi and Ugandan holidays snaps are present.  Just checked that : 
![image](https://github.com/spatie/holidays/assets/6143450/f7995f95-00dd-4699-883a-2838952c0e5a)

Reply:
Correct! I already fixed it on the main branch too 👍